sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/meta/provider/kafka/KafkaTableProviderThriftIT.java

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
Could you also add a test that will run on SQL precommit as well? You could use the same thrift file as in IT tests.
Example here: https://github.com/apache/beam/blob/master/sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/meta/provider/kafka/BeamKafkaTableAvroTest.java

Reply:
Sure, I'll add those.